functional/cypress-tests/cypress/support/widgetDynamicTab.js

```diff
@@ -1,154 +1,196 @@
-import Widgets from "../fixtures/Widgets";
+import Widgets from '../fixtures/Widgets'
 
 export function fillAemHealthcheck() {
-  let healthcheckKeys = Object.keys(Widgets.aemHealthcheck.healthChecks);
-  cy.get('[data-cy="widget-form-endpoint-input"]').click();
-  cy.get(`[data-value="${Widgets.aemHealthcheck.endpoint}"]`).click();
-  cy.fillSchedulePeriod(Widgets.aemHealthcheck.schedulePeriod);
-  cy.get('[data-cy="widget-form-selected-health-checks-input"]').click();
-  for (let i = 0; i < healthcheckKeys.length - 1; i++) {
-    let healthcheck = healthcheckKeys[i];
-    let label = Widgets.aemHealthcheck.healthChecks[healthcheck].label;
-    if (i == 0 || i == 3 || i == 11) {
-    } else if (i == 12) {
-      cy.contains("span", `${label}`).type("{esc}");
-    } else {
-      cy.contains("span", `${label}`).click();
+    const healthcheckKeys = Object.keys(Widgets.aemHealthcheck.healthChecks)
+    cy.get('[data-cy="widget-form-endpoint-input"]')
+        .click()
+    cy.get(`[data-value="${Widgets.aemHealthcheck.endpoint}"]`)
+        .click()
+    cy.fillSchedulePeriod(Widgets.aemHealthcheck.schedulePeriod)
+    cy.get('[data-cy="widget-form-selected-health-checks-input"]')
+        .click()
+    for (let i = 0; i < healthcheckKeys.length - 1; i++) {
+        const healthcheck = healthcheckKeys[i]
+        const label = Widgets.aemHealthcheck.healthChecks[healthcheck].label
+        if (i == 0 || i == 3 || i == 11) {
+        } else if (i == 12) {
```

Comment:
Maybe not necessarily connected to eslint changes, but do we have to keep empty if statement?